roles/nova-data/tasks/docker.yml

```diff
@@ -14,7 +12,28 @@
   until: result|success
   retries: 3
   delay: 60
+
+- name: pip install novadocker
+  pip: name='{{ nova.driver.docker.dest }}'
+       extra_args='-i {{ openstack.pypi_mirror }}'
+  register: result
+  until: result|success
+  retries: 3
+  delay: 60
+  when: result|success and openstack_install_method == 'source'
```

Comment:
where is this result coming from in this test? Seems weird to have a conditional like this, but maybe I'm missing context.